vimeo/vimeo-networking-java
# Api category localization

#### Issue
When requests are made, particularly with categories, we are supposed to be able to get a localized version back. On categories, this means a few titles will be in a different language. We need to tell the API that we want a different localization for a request.

#### Summary
Created a LanguageHeaderInterceptor to automatically add the `Accept-Language` header to HTTP requests made to the API. Done by passing in the user's locales to the Configuration Builder and then passing on the language codes to the interceptor. On devices that support multiple locales, a string of all the locales will be passed on and the API will select the first support locale. Defaults to English if none of the locales are supported.

#### How to Test
- Go to Explore tab and check the current language of the category titles.
- Close Vimeo app and change device language
- Open Vimeo app and go back to Explore tab and verify category titles have changed to new language.
